README.md

```diff
@@ -1,38 +1,31 @@
 # Developer Similarity
 
-This is a legacy from source{d}'s research in developer similarity once it was a [recruitment business](http://web.archive.org/web/20161020170426/http://sourced.tech/).
+This is a legacy from [source{d}](http://www.sourced.tech/)'s research in developer similarity when its main goal was to [match developers and companies](http://web.archive.org/web/20161020170426/http://sourced.tech/).
```

Comment:
how about "link developers with companies"? there are many "match"-es below

Reply:
Let's settle for connect, then.